code/model/SiteTree.php

```diff
@@ -1411,29 +1399,23 @@ public function MetaTags($includeTitle = true) {
 		return $tags;
 	}
 
-
 	/**
-	 * Returns the object that contains the content that a user would
-	 * associate with this page.
+	 * Returns the object that contains the content that a user would associate with this page.
 	 *
-	 * Ordinarily, this is just the page itself, but for example on
-	 * RedirectorPages or VirtualPages ContentSource() will return the page
-	 * that is linked to.
+	 * Ordinarily, this is just the page itself, but for example on RedirectorPages or VirtualPages ContentSource() will
+	 * return the page that is linked to.
 	 *
-	 * @return SiteTree The content source.
+	 * @return SiteTree The content source
```

Comment:
The spec allows all three, but I prefer `self` for `return $this` and `static` for instances of the same type/subtype that are not `$this`.